wordpress-mobile/gutenberg-mobile
# Change Aztec to not send changes notification when not user related

This PR changes the way Aztec is configured to avoid send notifications back when methods like `setHTML` are invoked.

This way we avoid callbacks to JS code when the change was originally sent by JS. 

To test:
 - Open the demo app
 - Test all text blocks to see if nothing is working correctly:
 - [x] Paragraph
 - [x] Quote
 - [x] Heading
 
Update release notes:
